qualtran/testing.py

```diff
@@ -567,6 +567,7 @@ def assert_bloq_example_serializes(bloq_ex: BloqExample) -> None:
         raise BloqCheckException.fail(
             f'Roundtrip equality failed.\n{bloq=}\n{bloq_roundtrip=}\n'
         ) from e
+    return BloqCheckResult.PASS, ''
```

Comment:
this isn't right, right? the annotation and docs say this should return None (like an assertion)

Reply:
Done.